pkg/apis/core/v1/conversion.go

```diff
@@ -463,3 +463,13 @@ func dropInitContainerAnnotations(oldAnnotations map[string]string) map[string]s
 	}
 	return newAnnotations
 }
+
+// Convert_core_PersistentVolume_To_v1_PersistentVolume is defined outside the autogenerated file for use by other API packages
+func Convert_core_PersistentVolume_To_v1_PersistentVolume(in *core.PersistentVolume, out *v1.PersistentVolume, s conversion.Scope) error {
```

Comment:
this should have moved the `Convert_core_PersistentVolumeSpec_To_v1_PersistentVolumeSpec` function

Reply:
Sorry about overlooking this. Fixed.